chart/tests/helm_template_generator.py

```diff
@@ -31,9 +32,12 @@
 
 BASE_URL_SPEC = "https://raw.githubusercontent.com/instrumenta/kubernetes-json-schema/master/v1.14.0"
 
```

Comment:
the concatenation of api version and kind in this way, `keda.sh/v1alpha1::ScaledObject` is a made-up syntax that only serves to provide a way to look up the url for a particular combination of  api version + kind